OperationalExcellenceAccNetworkingSample/Deployment/accNetwork.json

```diff
@@ -0,0 +1,495 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {
+        "accNetworkVnetName": "accNetwork-vnet",
+        "accNtwkStgAccountPrefix": "anstorage"
```

Comment:
Suggestion: Introduce a calculated variable to hold the ful name of the storage account instead of repeating the same formula everywhere. Something like:
    "variables": {
        "accNetworkVnetName": "accNetwork-vnet",
        **"accNywStgAccountName" : "[concat('anstorage', uniqueString(resourceGroup().id))]"**
    },